.eslintrc.js

```diff
@@ -25,6 +25,12 @@ module.exports = {
     // Disable error on devDependencies importing since this isn't a TS library
     'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
     'no-await-in-loop': 0,
+    'no-restricted-syntax': [
+      'off',
+      {
+        selector: 'ForOfStatement',
```

Comment:
This is the relevant syntax:
```ts
  for await (const { statusChange } of this.provider.subscriptions.statusChange({
      transactionId: this.id,
    }))
...
```
Airbnb doesn't like it (see [this](https://github.com/airbnb/javascript/issues/1271) and [this](https://github.com/airbnb/javascript/issues/1271)), but it's valid JS syntax that's also [supported](https://the-guild.dev/graphql/sse/recipes#with-asynciterator) by graphql-sse.

Reply:
Is there an alternative that wouldn't require disabling `eslint` rules?